src/bun.js/base.zig

```diff
@@ -2122,6 +2122,11 @@ pub export fn MarkedArrayBuffer_deallocator(bytes_: *anyopaque, _: *anyopaque) v
     // zig's memory allocator interface won't work here
     // mimalloc knows the size of things
     // but we don't
+    if (comptime Environment.allow_assert) {
```

Comment:
[existing bug reproduced by sveltekit build. this assertion makes it fail consistantly]